constants_1U.py

```diff
@@ -15,11 +15,11 @@
 #------------date format yyyy,mm,dd
 LINE1 = ('1 41783U 16059A   18093.17383152  .00000069  00000-0  22905-4 0  9992') #Insert TLE Here
 LINE2 = ('2 41783  98.1258 155.9141 0032873 333.2318  26.7186 14.62910114 80995') 
-
+Inclination = 98.1258

```

Comment:
I think it's better to extract this parameter from TLE. Otherwise it will create multiple definition for inclination, One in TLE and one here. High chances that people will change TLE and this will be missed. same for period